### PR TITLE
Update svelte-search, use new removeFormAriaAttributes option & alway…

### PR DESCRIPTION
**Pull request by Skovvart**
…s include ul, update activedescendant

More info on #18 and linked items

### CODE REVIEW

File: src/Typeahead.svelte [hunk 1]
```diff
@@ -125,12 +125,13 @@
...
   <Search
     {id}
+    removeFormAriaAttributes={true}
     {...$$restProps}
     bind:ref={searchRef}
     aria-autocomplete="list"
     aria-controls="{id}-listbox"
     aria-labelledby="{id}-label"
-    aria-activedescendant=""
+    aria-activedescendant={selectedIndex >= 0 && !hideDropdown ? `${id}-result-${selectedIndex}` : null}
```
Comment (metonym) [inline src/Typeahead.svelte:134]:
```suggestion
    aria-activedescendant={selectedIndex >= 0 && !hideDropdown && results.length > 0 ? `${id}-result-${selectedIndex}` : null}
```

There is an edge case where the `aria-activedescendant` has a value when the search field is cleared.

To reproduce:

- Search a value with results
- Clear the value by selecting the text and pressing backspace
- **Unexpected**: the `aria-activedescendant` is set to the first list item id

The solution would be to also check that the results length is greater than zero.

**Review by metonym — CHANGES_REQUESTED** [review]
This looks great. One requested change

**Comment by Skovvart** [comment]
> This looks great. One requested change

Change applied :-)

**Review by metonym — APPROVED** [review]
This is great work! Thank you for getting to the bottom of this

**Pull request merged**